### PR TITLE
dnssec support for zones

### PR DESCRIPTION
**Pull request by reinoud**
add dnssec support for zones

### CODE REVIEW

**Comment by mburtless** [comment]
Thanks for the contribution @reinoud!  This will be a great addition to the zones module.

Have you tested disabling dnssec on a zone using this PR?  In my own testing, the `dnssec` param always seems to evaluate to true, even when it is set to `false` in the playbook.

**Comment by reinoud** [comment]
Nice catch. Our businesscase is to always use dnssec, so I did not think about it :-)

I'll look into it

Reinoud

File: library/ns1_zone [hunk 1]
```diff
@@ -212,6 +218,7 @@ def main():
             networks        = dict(required=False, default=None),
             secondary       = dict(required=False, type='dict', default=None),
             primary         = dict(required=False, default=None),
+            dnssec          = dict(required=False, default=None),
```
Comment (mburtless) [inline library/ns1_zone]: This param's type should be boolean (`type='bool'`) as it does not take arbitrary string input

**Comment by mburtless** [comment]
This has been added the updated `ns1_zone` module in the `v2` branch, which will eventually replace the current modules in `master` and be submitted to ansible-core for inclusion in Ansible 2.9.

I'm going to close out this PR in favor of merging the `v2` branch to `master`.  Thanks for the contribution!

**Pull request closed without merging**